pkg/kubelet/cm/memorymanager/state/state_checkpoint_test.go

```diff
@@ -99,81 +99,24 @@ func TestCheckpointStateRestore(t *testing.T) {
 				},
 			},
 		},
-		// {
-		// 	"Restore valid checkpoint",
-		// 	`{
-		// 		"policyName": "none",
-		// 		"defaultCPUSet": "1-3",
-		// 		"entries": {
-		// 			"pod": {
-		// 				"container1": "4-6",
-		// 				"container2": "1-3"
-		// 			}
-		// 		},
-		// 		"checksum": 3610638499
-		// 	}`,
-		// 	containermap.ContainerMap{},
-		// 	"",
-		// 	&stateMemory{
-		// 		assignments: ContainerMemoryAssignments{
-		// 			"pod": map[string][]Block{
-		// 				"container1": []Block{
-		// 					{
-		// 						NUMAAffinity: 0,
-		// 						Type:     core.ResourceMemory,
-		// 						Size:     1024,
-		// 					},
-		// 				},
-		// 			},
-		// 		},
-		// 		machineState: MemoryMap{
-		// 			0: map[core.ResourceName]MemoryTable{
-		// 				core.ResourceMemory: MemoryTable{
-		// 					Allocatable:    1024,
-		// 					Free:           1024,
-		// 					Reserved:       512,
-		// 					SystemReserved: 512,
-		// 					TotalMemSize:   2048,
-		// 				},
-		// 			},
-		// 		},
-		// 	},
-		// },
-		// {
-		// 	"Restore checkpoint with invalid checksum",
-		// 	`{
-		// 		"policyName": "none",
-		// 		"defaultCPUSet": "4-6",
-		// 		"entries": {},
-		// 		"checksum": 1337
-		// 	}`,
-		// 	containermap.ContainerMap{},
-		// 	"checkpoint is corrupted",
-		// 	&stateMemory{},
-		// },
-		// {
-		// 	"Restore checkpoint with invalid JSON",
-		// 	`{`,
-		// 	containermap.ContainerMap{},
-		// 	"unexpected end of JSON input",
-		// 	&stateMemory{},
-		// },
-		// {
-		// 	"Restore checkpoint with unparsable assignment entry",
-		// 	`{
-		// 		"defaultCPUSet": "1-3",
-		// 		"entries": {
-		// 			"pod": {
-		// 				"container1": "4-6",
-		// 				"container2": "asd"
-		// 			}
-		// 		},
-		// 		"checksum": 962272150
-		// 	}`,
-		// 	containermap.ContainerMap{},
-		// 	`could not parse cpuset "asd" for container "container2" in pod "pod": strconv.Atoi: parsing "asd": invalid syntax`,
-		// 	&stateMemory{},
-		// },
+		{
+			"Restore checkpoint with invalid checksum",
+			`{
+				"machineState":{"0":{"memory":{"total":2048,"systemReserved":512,"allocatable":1024,"reserved":512,"free":1024}}},
```

Comment:
Could we also do it realistic?
i.e. `allocatable = total - systemReserved` (allocatable should be `1536`)

Reply:
ups missed it, done